test/testLocaleInfo.js

```diff
@@ -6116,7 +6116,7 @@ module.exports.testlocaleinfo = {
         var info = new LocaleInfo("ks-Arab-IN");
         test.ok(info !== null);
 
-        test.equal(info.getCurrencyFormats().common, "{s} {n}");
+        test.equal(info.getCurrencyFormats().common, "{s}{n}");
```

Comment:
seems it is cldr changed from cldr 40 to 41. Probbally, tcs in iLib need to udpate.

Reply:
I submitted PRs for cldr 41 a few months back in the ilib project, remember?

What does "tcs" stand for?

Reply:
I meant test case here. : https://github.com/iLib-js/iLib/blob/development/js/test/root/testlocaleinfo.js#L6211
I think iLib haven't updated all feature to cldr41. I saw only some feature update PRs. no?